src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/__init__.py

```diff
@@ -30,9 +37,32 @@ def transform_extension_list_available(results):
             return [OrderedDict([('Name', r)]) for r in results]
 
         def validate_extension_add(namespace):
+            if IS_WINDOWS == False:
+                try:
+                    logger.debug('Reading from: %s', LIST_FILE_PATH)
```

Comment:
Prefix all log messages with 'Linux distro check: ' so we know where the logs are coming from.
Thanks for using the logger though!

Reply:
done